bus-mapping/src/circuit_input_builder.rs

```diff
@@ -132,6 +144,8 @@ pub struct CircuitInputBuilder<C: CircuitsParams> {
     pub code_db: CodeDB,
     /// Block
     pub block: Block,
+    /// Chunk
+    pub chunks: Vec<Chunk>,
```

Comment:
It seems to me that we use `chunk_ctx.cur` to access `chunks` vector but still better document it somewhere